packages/jsx-email/src/renderer/conditional.ts

```diff
@@ -0,0 +1,89 @@
+import type { Content, Element, Literal, Parents, Root } from 'hast';
+
+// Model rehype `raw` node locally for strong typing.
+interface Raw extends Literal {
```

Comment:
please do not duplicate code that's already available from other files. import types Raw and ParentWithRaw from render/raw.ts